.github/workflows/python-app.yml

```diff
@@ -30,6 +30,10 @@ jobs:
     - name: Lint with flake8
       run: |
         poetry run flake8 ./src
+    - name: Lint with pylint
+      run: |
+        poetry run pylint ./src
```

Comment:
Would it be appropriate to add a sentence or two about pylint to the [Code Style section](https://github.com/DistributedProofreaders/guiguts-py/blob/master/README.md#code-style) of the README, similar to those about black and flake8?

Reply:
@windymilla  I updated that section of the README; let me know what you think of the new wording.

Reply:
Thanks - very happy with that